sphinx/ext/autodoc.py

```diff
@@ -1520,6 +1528,7 @@ def setup(app):
     app.add_config_value('autodoc_member_order', 'alphabetic', True)
     app.add_config_value('autodoc_default_flags', [], True)
     app.add_config_value('autodoc_docstring_signature', True, True)
+    app.add_config_value('autodoc_dumb_docstring', False, True)
```

Comment:
Suggestion:  This is not so much a “dumb” version but, maybe, a *verbatim* one?  Also, it doesn't apply to the docstring as a whole but to the default values, ie. the argspec/signature.